mysqlTransformer.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import json
+import urllib.request
+import io
+import gzip
+
+from dataTransformations import Transformations
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark import SparkConf
+from pyspark import SparkFiles
+
+# Takes arguments: Spark master, Cassandra host, Minio host, path of the files
+sparkMaster = sys.argv[1]
+cassandraHost = sys.argv[2]
+minioHost = sys.argv[3]
+filePath = sys.argv[4]
+
+conf = SparkConf() \
+    .setAppName("PySpark Cassandra Test") \
+    .setMaster(sparkMaster) \
+    .set("spark.cassandra.connection.host", cassandraHost)
+
+sc = CassandraSparkContext(conf=conf)
+
+confPath = SparkFiles.get("conf.json")
+with open(confPath) as f:
+    conf = json.load(f)
+
+res = urllib.request.urlopen("http://"+minioHost+"/"+filePath+"_"+conf["src_table"]+".gz")
+compressed = io.BytesIO(res.read())
+decompressed = gzip.GzipFile(fileobj=compressed)
+lines = decompressed.readlines()
+data = sc.parallelize(lines)
+
+indexes = {}
+print(data.first())
+schema = data.first().decode().split(",")
+
+for i in range(len(schema)):
+    for c in conf["column_mapping"]:
+        if schema[i].replace('"', '') == c:
+            indexes[schema[i].replace('"', '')] = i
+
+print(schema)
+print(indexes)
+
+def createDic(a):
```

Comment:
Add some comments in the following code, so that I can better understand it and give you feedback to improve it understandability.